src/coreComponents/managers/initialization.cpp

```diff
@@ -181,7 +181,7 @@ void setupCaliper()
   std::int64_t const numThreads = 1;
   adiak::value( "OpenMP", "Off" );
 #endif
-  pushStatsIntoAdiak( "numThreads", numThreads );
+  pushStatsIntoAdiak( "numThreads", static_cast<int>(numThreads) );
```

Comment:
I would change `numThreads` to be an `int` instead of adding the `static_cast`. That said `pushStatsIntoAdiak` will calculate the sum of `numThreads` across all ranks and push that into Adiak. This isn't a problem for `numThreads` but it might be for other stats such as the total number of nodes or elems where we have run problems with a few billion elements.